modules/fts/pages/fts-scoring-tf-idf-how-it-works.adoc

```diff
@@ -0,0 +1,67 @@
+= tf-idf
+
+[abstract]
+`tf-idf`, a short form of *term frequency-inverse document frequency*, is a numerical statistical value that is used to reflect how important a word is to a document in collection or scope. 
+
+`tf-idf` is used as a weighting factor in a search for information retrieval and text mining. The `tf–idf` value increases proportionally to the number of times a word appears in the document, and it is offset by the number of documents in the collection or scope that contains the word.
+
+Search engines often uses the validations of `tf-idf` weighting scheme as a tool in scoring and ranking a document's relevance for a given query.
+
+`tf–idf` can also be used for stop-words filtering in various subject fields, including text summarization and classification.
+
+FTS’s internal text indexing library (bleve) uses a slightly modified version of standard `tf-idf` scoring. This improvisation is done to normalize the score by various relevant factors. The search scoring happens at query time.
```

Comment:
I think this is an unimportant detail.  All Couchbase users care about is the score, not Bleve.

Reply:
Update: Remove line 12